frontend/test/playwright/utils/expect-snapshot.ts

```diff
@@ -89,10 +89,13 @@ export const expectSnapshot: ExpectSnapshot = async (
     .soft(await screenshotAble.screenshot(screenshotOptions))
     .toMatchSnapshot(getSnapshotName(name, "light"), snapshotOptions)
 
-  if (!(useColorMode === true)) {
+  if (!useColorMode) {
     return
   }
+
   await turnOnDarkMode(page, dir ?? "ltr")
+  // Wait for the theme to change.
+  await sleep(100)
 
   // Wait for the theme to change.
   await sleep(100)
```

Comment:
Is it intentional to have two sleeps? It's unfortunate to need to use one at all... could we instead do something in `turnOnDarkMode` that probes the page for the expected classname on the root element?

If we absolutely need two separate sleeps, then it would be better to document them explicitly in reference to each other (and maybe move them into `turnOnDarkMode`, if it's always needed anyway).

Reply:
The two sleeps are caused by the faulty merge conflict resolution :( I'm removing the extra one.
The existence of class name does not guarantee that the rendering has caught up and all colors have changed. I've seen a range of white-to-gray colors in the separator lines in the flaky snapshots when working on this, so I think a sleep is the best solution here.

Reply:
Okay, understood.

I'm surprised it doesn't happen in a single rendering pass :thinking:. What causes the browser to take multiple passes to render in response to the classname change, I wonder.